docs/config_guide.md

```diff
@@ -1,7 +1,8 @@
 # Configuration Guide
+
 All configuration is stored in the `config` directory located in the project root.
 
-Secrets and configuration variables that are likely to change between deployment environments are stored in a file called `config.env`. This file should *NOT* be checked into source control.
+Secrets and configuration variables that are likely to change between deployment environments are stored in a file called `config.env`. This file should _NOT_ be checked into source control.
```

Comment:
Looks like someone hasn't stopped prettier from linting markdown files 🤔 lol

Reply:
🙈 I turned it off half way through this PR